server/src/main/java/org/elasticsearch/index/query/MatchBoolPrefixQueryBuilder.java

```diff
@@ -377,6 +377,6 @@ public String getWriteableName() {
 
     @Override
     public TransportVersion getMinimalSupportedVersion() {
-        return TransportVersions.V_7_2_0;
+        return TransportVersions.ZERO;
```

Comment:
Just a check: this is now "ZERO" to mean "I'm fine with whatever TV this cluster might have" (which is all TV supported by 8.18+), right?

Reply:
Yup - it's used in several other places too